packages/nvidia-container-toolkit/nvidia-container-toolkit-config-ecs.toml

```diff
@@ -3,3 +3,6 @@ root = "/"
 path = "/usr/bin/nvidia-container-cli"
 environment = []
 ldconfig = "@/sbin/ldconfig"
+
+[nvidia-container-runtime]
+mode = "cdi"
```

Comment:
Question - per the description 
> This truly enables CDI, and drops the dependency on the prestart hook provided by nvidia-container-toolkit

I am trying to connect this with https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/458/files#diff-bf719304fec29a1f2736a935447edf3308cd718549ba13130f2166377c1c6e1b which already dropped the pre-start hook injection from nvidia-container-toolkit. Do you mean nvidia-container-runtime here?

Reply:
The change you linked didn't drop the dependency on the pre-start hook, it only dropped the dependency on shimpei and oci-add-hooks to inject the prestart hook. The change that you linked still depended on the prestart hook. With this change, at least for ECS, the prestart hook is no longer needed.